README.md

```diff
@@ -6,7 +6,7 @@
 
 ### Contributors
 
-- Classmates: Dina Ayoub, Simon Panek
+- Classmates: Simon Panek, Tina Myers, Mariko Alvarado
 - Code Fellows TAs: N/A
```

Comment:
Remember to include what is needed in your .env and if you have ports hard coded into driver and vendor then make sure to make a note of that in your README.